### PR TITLE
add vit_pretrained.py and test_vit_pretrained.py

### PR DESCRIPTION
**Pull request by oneseer**
add vit_pretrained.py for loading pretrained transformer.
add ut for loading pretrained.

### CODE REVIEW

File: towhee/trainer/models/vit/vit_pretrained.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    drop_head_variant
+)
+
+
+PRETRAINED_MODELS = {
+    'B_16': {
+      'config': get_b16_config(),
+      'num_classes': 21843,
+      'image_size': (224, 224),
+      'url': 'https://github.com/lukemelas/PyTorch-Pretrained-ViT/releases/download/0.0.2/B_16.pth'
```
Comment (junjiejiangjjj) [inline towhee/trainer/models/vit/vit_pretrained.py:34]: Why use this URL? What to do if the link fails

Comment (oneseer) [reply towhee/trainer/models/vit/vit_pretrained.py:34]: when towhee hub is ready,  the open sourced models will be mirrored in our hub

Comment (junjiejiangjjj) [reply towhee/trainer/models/vit/vit_pretrained.py:34]: #98

**Review by junjiejiangjjj — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**